lib/engine/round/g_1817/acquisition.rb

```diff
@@ -5,34 +5,22 @@
 module Engine
   module Round
     module G1817
-      class Acquisition < Merger
+      class Acquisition < Round::Merger
```

Comment:
should this be a merger or an auction?

Reply:
Merger fits more, since you'll have to select the company to merge with.